content/posts/auto-sync/index.md

```diff
@@ -464,36 +471,28 @@ The same structure applies to the printing of the asm text. Though we have only
 
 ## Wrap up
 
-If one looks at the update whole procedure, it is still a rather complicated number of steps to take. But the result is worth it.
-The amount of time someone has to spend for updating an architecture module in Capstone went down from "no one did it" to roughly 4-19 hours.
+If one looks at the whole update procedure, it is still a rather complicated. But the result is worth it.
+
+The amount of time someone has to spend for updating an architecture module in Capstone went down from "no one did it" to roughly 6-29 hours.
 To update the ARM architecture module to LLVM 16 for example, the times were:
 
 1. Rebasing patched backends to new LLVM release = ~1-3h
 2. Running the update scripts and diffing = 5min - 1h
 3. Fixing rest of build errors by hand = ~30min - 5h
 4. Handle new operands on the CS side (filling the detail info and tests) - 3-10h
+5. Bug fixing - 2h-10h
 
-(Please be aware though, that the time estimates from above don't include the "read into" time someone has to spend.)
+(Please be aware though, that the time estimates from above don't include the "read into" time someone has to spend).
 
 The lower estimates are for small changes on the LLVM side, the upper for many changes (spanning multiple LLVM releases).
 
-Here is the whole process in action:
```

Comment:
You decided to remove the asciicast?

Reply:
Because it was no longer representative how it works.
And with the many changes, the Differ doesn't work too good anymore.

But you are right, let me do a new one.